### DISCUSSION TITLE
Are my Iota lost?

### DISCUSSION THREAD

**Discussion by muttaphuckinjones** [iotaledger/legacy-migration-tool#134] [Help & Support] [answered]
Hello, I tried to migrate my old Iota. Everything worked, except I was not able to save a Stronghold Backup for whatever reason. I was not able to click the button, nothing happened. So I saved the pdf Recovery Sheet and I thought the recovery words would be filled in. Unfortunately I didn't check the pdf and just clicked all the 24 words randomly through. After that I wanted to get my Shimmer for my Iota. But now I see the recovery pdf is totaly empty and I was supposed to print it out and fill the words in by myself. Am I right that all of my Iota are lost now without a Stronghold file and without the 24 recovery words? :'( Daamn, I'm dumb

**Comment by begonaalvarezd** [comment, marked as answer]
Maybe not everything is lost, if you didnt uninstall the tool completely yet, there might still be a stronghold backup saved where the tool was installed. Look for a `.stronghold` inside
- Windows: `%APPDATA%\Roaming\/IOTA\ Legacy\ Migration\ Tool\__storage__/[profileID]`
- MacOS: `$HOME/Library/Application\ Support/IOTA\ Legacy\ Migration\ Tool/__storage__/[profileID]`
- Linux: `~/.config//IOTA\ Legacy\ Migration\ Tool/__storage__/[profileID]`

Note that [profileID] is like an alphanumeric id/hash, just look for a .stronghold file somewhere inside `__storage__`

🤞🏼 good luck

**Reply by muttaphuckinjones** [reply]
Thank you very much my friend :) I'm so happy I asked here. Indeed there was a stronghold file in the directory you mentioned and now I was able to log into Firefly and see my IOTA Balance. Awesome!!! Thank you so much
